tardis/tests/tests_slow/conftest.py

```diff
@@ -1,9 +1,85 @@
 import os
+import shutil
+import tempfile
+import yaml
 import numpy as np
 import pytest
 from astropy import units as u
+from astropy.tests.helper import remote_data
 import tardis
 
+# For specifying error while exception handling
+from socket import gaierror
+
+try:
+    import dokuwiki
+except ImportError:
+    dokuwiki_available = False
+else:
+    dokuwiki_available = True
+
+
+def pytest_configure(config):
+    integration_tests_configpath = config.getvalue("integration-tests")
+    if integration_tests_configpath is not None:
+        integration_tests_configpath = os.path.expandvars(
+            os.path.expanduser(integration_tests_configpath)
+        )
+        config.option.integration_tests_config = yaml.load(
+            open(integration_tests_configpath))
+
+    # A common tempdir for storing plots / PDFs and other slow test related data
+    # generated during execution.
+    tempdir_session = tempfile.mkdtemp()
+    config.option.tempdir = tempdir_session
+    html_file = tempfile.NamedTemporaryFile(delete=False)
+    # Html test report will be generated at this filepath by pytest-html plugin
+    config.option.htmlpath = html_file.name
+
+
+@remote_data
+def pytest_unconfigure(config):
+    # Html report created by pytest-html plugin is read here, uploaded to
+    # dokuwiki and finally deleted.
+    if dokuwiki_available:
```

Comment:
You added a check to `pytest_configure` to check if we are running the integration tests or not. Please do the same for `pytest_unconfigure`